data/json/recipes/armor/other.json

```diff
@@ -34,19 +34,64 @@
   },
   {
     "result": "gold_bracelet",
-    "copy-from": "ring",
     "type": "recipe",
+	"category": "CC_ARMOR",
```

Comment:
Tab.

Reply:
All tabs have been replaced with the proper 4 spaces.